src/sentry/api/helpers/group_index/validators/group.py

```diff
@@ -4,7 +4,8 @@
 
 from sentry.api.fields import ActorField
 from sentry.models import Actor, Team, User
-from sentry.models.group import STATUS_UPDATE_CHOICES, SUBSTATUS_UPDATE_CHOICES
+from sentry.models.group import STATUS_UPDATE_CHOICES
+from sentry.types.group import SUBSTATUS_UPDATE_CHOICES
```

Comment:
Yeah, I don't like that split either - I can look at rehoming the GroupStatus + related types in a followup PR. Had to move the GroupSubStatus because of a circular dependency